### PR TITLE
docs: add documentation

### PR DESCRIPTION
**Pull request by fqutishat**
part #20

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,3 +1,13 @@
 [![Build Status](https://dev.azure.com/trustbloc/edge/_apis/build/status/trustbloc.agent-ui?branchName=master)](https://dev.azure.com/trustbloc/edge/_build/latest?definitionId=32&branchName=master)
 
 # agent-ui
+
+Agent UI is a javascript project that use VUE framework to create common js components
```
Comment (troyronda) [inline README.MD]: Vue 

https://vuejs.org/v2/guide/

Comment (troyronda) [inline README.MD]: missing . at end of sentence.

File: docs/test/test.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Agent UI - Test
+
+## Prerequisites (General)
+
+- VUE js
```
Comment (troyronda) [inline docs/test/test.md]: Vue.js

File: README.MD [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+# Components
+
+- Invitation
```
Comment (troyronda) [inline README.MD]: are you linking to a description of this component?

Comment (fqutishat) [reply README.MD]: I will do it now

File: README.MD (hunk 1 above)
Comment (fqutishat) [reply README.MD]: done

Comment (fqutishat) [reply README.MD]: done

File: docs/test/test.md (hunk 2 above)
Comment (fqutishat) [reply docs/test/test.md]: done

File: README.MD (hunk 3 above)
Comment (fqutishat) [reply README.MD]: done

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+[![Build Status](https://dev.azure.com/trustbloc/edge/_apis/build/status/trustbloc.agent-ui?branchName=master)](https://dev.azure.com/trustbloc/edge/_build/latest?definitionId=32&branchName=master)
+
+# agent-ui
+
+Agent UI is a javascript project that use [Vue](https://vuejs.org/v2/guide/) framework to create common js components.
```
Comment (troyronda) [inline README.md]: The link to Vue is probably just https://vuejs.org

File: docs/components/invitation.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Invitation
+
+Invitation is vue component to create and accept [DID exchange invitation](https://github.com/hyperledger/aries-rfcs/tree/master/features/0023-did-exchange#0-invitation-to-exchange)
```
Comment (troyronda) [inline docs/components/invitation.md]: missing 'a' between is and vue.

File: docs/components/invitation.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+# Html tags
```
Comment (troyronda) [inline docs/components/invitation.md]: HTML

File: README.md (hunk 4 above)
Comment (fqutishat) [reply README.md]: done

File: docs/components/invitation.md (hunk 5 above)
Comment (fqutishat) [reply docs/components/invitation.md]: done

File: docs/components/invitation.md (hunk 6 above)
Comment (fqutishat) [reply docs/components/invitation.md]: done

**Pull request merged**